.pipelines/cni/lsg/lsg-cni-intergration-template.yaml

```diff
@@ -49,43 +43,76 @@ stages:
-              addSpnToEnvironment: true
-              inlineScript: |
-                echo cns version - $(cnsVersion)
-                echo cni version - $(cniVersion)
+      - ${{ if contains(parameters.name, 'linux') }}:
+        - job: integration
+          displayName: "Integration Test - ${{ parameters.name }}"
+          steps:
+            - task: AzureCLI@1
+              inputs:
+                azureSubscription: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
+                scriptLocation: "inlineScript"
+                scriptType: "bash"
+                addSpnToEnvironment: true
+                inlineScript: |
+                  echo cns version - $(cnsVersion)
+                  echo cni version - $(cniVersion)
 
-                echo "Start Integration Tests on Overlay Cluster"
-                make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(commitID)
-                kubectl cluster-info
-                kubectl get po -owide -A
-                sudo -E env "PATH=$PATH" make test-integration CNS_VERSION=$(cnsVersion) CNI_VERSION=$(cniVersion) INSTALL_CNS=true INSTALL_AZURE_CNI_OVERLAY=true CNS_IMAGE_REPO=$(CNS_IMAGE_REPO) CNI_IMAGE_REPO=$(CNI_IMAGE_REPO)
-            name: "overlaye2e"
-            displayName: "Overlay Integration"
-            retryCountOnTaskFailure: 2
+                  echo "Start Integration Tests on Overlay Cluster"
+                  make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(commitID)
+                  kubectl cluster-info
+                  kubectl get po -owide -A
+                  sudo -E env "PATH=$PATH" make test-load CNS_ONLY=true CNS_VERSION=$(cnsVersion) CNI_VERSION=$(cniVersion) INSTALL_CNS=true INSTALL_AZURE_CNI_OVERLAY=true CNS_IMAGE_REPO=$(CNS_IMAGE_REPO) CNI_IMAGE_REPO=$(CNI_IMAGE_REPO)
+              name: "overlaye2e"
+              displayName: "Overlay Integration"
+              retryCountOnTaskFailure: 2
+      - ${{ if contains(parameters.name, 'cilium') }}:
+        - job: integration
+          displayName: "Deploy Cilium Components"
+          steps:
+            - task: AzureCLI@1
+              displayName: "Install Cilium, CNS, and ip-masq-agent"
+              retryCountOnTaskFailure: 2
+              inputs:
+                azureSubscription: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
+                scriptLocation: "inlineScript"
+                scriptType: "bash"
+                addSpnToEnvironment: true
+                inlineScript: |
+                  set -ex
+                  az extension add --name aks-preview
+                  make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(commitID)
+                  ls -lah
+                  pwd
+                  kubectl cluster-info
+                  kubectl get po -owide -A
+                  echo "deploy Cilium ConfigMap"
+                  kubectl apply -f test/integration/manifests/cilium/cilium-config.yaml
```

Comment:
you'll need to rebase and use the new paths based on cilium version to pull the correct templates

Reply:
ex: `kubectl apply -f test/integration/manifests/cilium/v${DIR}/cilium-config/cilium-config.yaml`